### PR TITLE
Upgrade to using netbox-5.0.0-beta.161

### PR DESCRIPTION
**Pull request by henrybear327**
Upgrade to the latest version, to reduce migration headache in the future

### CODE REVIEW

File: kind/local-env.sh [hunk 1]
```diff
@@ -48,7 +48,7 @@ helm upgrade --install --namespace="${NAMESPACE}" netbox \
   --set externalDatabase.existingSecretName="netbox.netbox-db.credentials.postgresql.acid.zalan.do" \
   --set externalDatabase.existingSecretKey="password" \
   --set redis.auth.password="password" \
-  https://github.com/netbox-community/netbox-chart/releases/download/netbox-5.0.0-beta.34/netbox-5.0.0-beta.34.tgz
+  https://github.com/netbox-community/netbox-chart/releases/download/netbox-5.0.0-beta.160/netbox-5.0.0-beta.160.tgz
```
Comment (alexandernorth) [inline kind/local-env.sh:51]: latest just changed to 161: https://github.com/netbox-community/netbox-chart/releases/tag/netbox-5.0.0-beta.161

Comment (henrybear327) [reply kind/local-env.sh:51]: LOL, on it

**Comment by henrybear327** [comment]
Removed the accidental commit for the netbox postgres `database.sql` dump

**Review by alexandernorth — APPROVED** [review]
lgtm

**Pull request merged**